AdobeStockImage/Model/ResourceModel/Keyword/Collection.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+namespace Magento\AdobeStockImage\Model\ResourceModel;
```

Comment:
I believe this file has to be removed from the pull request

Reply:
Deleted